app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/your-organization.gjs

```diff
@@ -1,18 +1,56 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { on } from "@ember/modifier";
 import { action } from "@ember/object";
 import DButton from "discourse/components/d-button";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 import i18n from "discourse-common/helpers/i18n";
 
 export default class AdminConfigAreasAboutYourOrganization extends Component {
+  @tracked showSavedAlert = false;
+
+  companyName = this.args.yourOrganization.companyName.value;
+  governingLaw = this.args.yourOrganization.governingLaw.value;
+  cityForDisputes = this.args.yourOrganization.cityForDisputes.value;
+
+  @action
+  onCompanyNameChange(event) {
+    this.companyName = event.target.value;
+  }
+
+  @action
+  onGoverningLawChange(event) {
+    this.governingLaw = event.target.value;
+  }
+
+  @action
+  onCityForDisputesChange(event) {
+    this.cityForDisputes = event.target.value;
+  }
```

Comment:
In all those cases you can use this in the template instead:

```hbs
{{on "input" (withEventValue (fn (mut this.companyName)))}}
```

Reply:
So is this what should be used at all times now in Glimmer components? Should `companyName` and the others be @tracked or not? Is this the replacement for the "classic" 2 way data binding? It's a bit of a moot point with FormKit upcoming I guess but it would be good to have guidance on this.

Reply:
Yes, two-way bindings are out. So you should use either `withEventValue` helper or define actions (if you need to transform data in some way)

re: `@tracked` - it depends if you use those values to render anything, directly or indirectly - through other properties that read them. (that's what this decorator pretty much does - "please update the UI whenever this changes")

I don't know yet how exactly it will look like with form-kit (still have to review it 😅) but I'd say those apply generally in most parts of the app.